avocado/core/spawners/podman.py

```diff
@@ -71,3 +71,9 @@ def spawn_task(self, task):
                                                          stdout=asyncio.subprocess.PIPE,
                                                          stderr=asyncio.subprocess.PIPE)
         yield from proc.wait()
+
+    def create_output_dir_for_task(self, task):
+        """In this case is a remote folder."""
+        local_dir = '/tmp/avocado/tasks/{}/output'.format(task.identifier)
```

Comment:
I am really confused here. You're creating a directory locally, and then asking the task to create it... is this intended to be the destination mount point for a `-v` usage?

Reply:
Sorry, my bad, this is a leftover of a rebase. I will remove this method. Is not being used anymore.